apps/frontend/src/components/questionary/questionaryComponents/Boolean/QuestionBooleanForm.tsx

```diff
@@ -25,6 +26,7 @@ export const QuestionBooleanForm = (props: QuestionFormProps) => {
     >
       {() => (
         <>
+          <TemplateEdit pageType="Question" />
```

Comment:
Can we have the naming be clear it is just a label component. Also do we have to hard code this as question or template is the information not being loaded in the parent component.